lte/gateway/c/oai/tasks/mme_app/mme_app_itti_messaging.c

```diff
@@ -235,6 +235,11 @@ int mme_app_send_s11_create_session_req(
   } else {
     session_request_p->msisdn.length = 0;
   }
+
+  // Fill User Location Information
+  session_request_p->uli.present = 0;   // initialize the presencemask
```

Comment:
Not sure if I'm reading this PR correctly, but why is this value always set to 0?

Reply:
Oh nvm! It is optionally set in `mme_app_get_user_location_information`?

Reply:
Yes, the presencemask is just initialized here to 0